src/Microsoft.Health.Fhir.Core/Models/SearchParameterInfo.cs

```diff
@@ -95,5 +95,7 @@ public SearchParameterInfo(string name)
         public bool IsPartiallySupported { get; set; }
 
         public IReadOnlyList<SearchParameterComponentInfo> Component { get; }
+
+        public IReadOnlyList<SearchParameterInfo> ResolvedComponents { get; set; } = Array.Empty<SearchParameterInfo>();
```

Comment:
Can we add comments to this properites?
<summary>
All possible components of composite search parameters. Should be present only if Type = SearchParamType.Composite
</summary>
 public IReadOnlyList<SearchParameterComponentInfo> Component { get; }

<summary>
Populated by query composite search parameters
</summary>
public IReadOnlyList<SearchParameterInfo> ResolvedComponents { get; set; } = Array.Empty<SearchParameterInfo>();